src/containers/MoviesContainer/MoviesContainer.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react'
+import './MoviesContainer.scss'
+import { getMovies } from '../../apiCalls/apiCalls'
+import { addMovies } from '../../actions';
+import { connect } from 'react-redux';
+import SmallMovieCard from '../../components/SmallMovieCard/SmallMovieCard'
+
+class MoviesContainer extends Component {
+  constructor(props) {
```

Comment:
Do we need to add props to the constructor if we aren't passing props through from App? I'm genuinely unsure. We can leave it for now but we might want to check again later to see if removing it has any effect.

Reply:
But It still get props from redux store.  That's why it should be written.